packages/metro/src/DeltaBundler/__tests__/DeltaTransformer-test.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
```

Comment:
Can you add the license section at the top of the file (Plus the @format and @emails annotations), like for example https://github.com/facebook/metro/blob/master/packages/metro/src/__tests__/index-test.js#L1-L12 ?

Reply:
Done; thanks, @rafeca